Project.toml

```diff
@@ -1,10 +1,11 @@
 name = "POMDPFiles"
 uuid = "9cf5b727-2e06-5671-8c87-8c6b0f729d5d"
 repo = "https://github.com/JuliaPOMDP/POMDPFiles.jl"
-version = "0.2.3"
+version = "0.2.4"
```

Comment:
I think this adds functionality in a backwards compatible manner, and so I think you would increment the minor version, not the patch version. See https://semver.org/

Reply:
Incrementing the minor version would make this `0.3.0`.

Reply:
(resolved in email)